discovery_engine_core/core/test-fixtures/articles-some-malformed-media-urls.json

```
@@ -0,0 +1,48 @@
+[
+  {
+    "title": "How to teach children about climate change, inspire hope and take action to change the\u00a0future",
+    "author": "Anonymous",
+    "published_date": "2022-01-27 19:30:00",
+    "published_date_precision": "timezone unknown",
+    "link": "https://xayn.com",
+    "clean_url": "xayn.com",
+    "excerpt": "Some children and youth find the effects of climate change are traumatic. Taking a trauma-informed approach to education can nurture resilience.",
+    "summary": "Some children and youth find the effects of climate change are traumatic. Taking a trauma-informed approach to education can nurture resilience.",
+    "rights": "xayn.com",
+    "rank": 361,
+    "topic": "news",
+    "country": "AU",
+    "language": "en",
+    "authors": [
+      "Anonymous"
+    ],
+    "media": "",
+    "is_opinion": false,
+    "twitter_account": "@XaynHQ",
+    "_score": 17.197401,
+    "_id": "4e30f7eaa51090aa96a91fcce2dd1ade"
+  },
+  {
+    "title": "Businesses \u2018more concerned than ever'",
+    "author": "Anonymous",
+    "published_date": "2022-01-23 13:01:00",
+    "published_date_precision": "full",
+    "link": "https://xayn.com",
+    "clean_url": "xayn.com",
+    "excerpt": "Australian business leaders are more concerned about climate change than ever before, with a new report finding there has been a significant shift in opinion in just eight months.",
+    "summary": "Australian business leaders are more concerned about climate change than ever before, with a new report finding there has been a significant shift in opinion in just eight months.",
+    "rights": "xayn.com",
+    "rank": 570,
+    "topic": "tech",
+    "country": "AU",
+    "language": "en",
+    "authors": [
+      "Anonymous"
+    ],
+    "media": "a88260009773cd_hero-phone.png",
+    "is_opinion": false,
+    "twitter_account": "@XaynHQ",
+    "_score": 17.118832,
+    "_id": "b7ecc49a15bc38bf224ae01556e52641"
+  }
+]
```

github wants to have a newline
```suggestion
]

```